### PR TITLE
feat: UI login and signup page

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Created UI for Login and Signup
Functionality is not added for now

image

image

### CODE REVIEW

File: src/pages/auth/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import Login from "../../components/auth/Login";
+
+const LoginPage = () => {
+  return (
+    <>
+      <Login />
+    </>
+  );
+};
+
```
Comment (JantuDeb) [inline src/pages/auth/LoginPage.js:11]: You can directly write the html here as you are not reusing <Login> component

File: src/pages/auth/SignupPage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Signup from "../../components/auth/Signup";
+
+const SignupPage = () => {
+  return (
+    <>
+      <Signup />
+    </>
+  );
```
Comment (JantuDeb) [inline src/pages/auth/SignupPage.js:9]: same here

**Review by JantuDeb — APPROVED** [review]
Overall code looks fine

**Review by tushartiwari7 — APPROVED** [review]
Code looks fine to me.
Suggestion: Can eliminate `forgot password`  checkbox if you are not planning to implement it.

File: src/components/auth/Login.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          <label for="password">Password</label>
+          <input type="password" />
+        </div>
+        <div class="flex justify-between align-center">
+          <div class="flex align-center">
+            <input type="checkbox" name="remember" id="remember" />
+            <label for="remember">Remember me</label>
+          </div>
+          <a href="#">Forgot Password?</a>
+        </div>
```
Comment (fahd23) [inline src/components/auth/Login.js:27]: You can remove this anchor tag if you are not using it

**Review by fahd23 — APPROVED** [review]
Code looks good to merge

**Pull request merged**